utility.cake

```diff
@@ -66,7 +66,7 @@ static int ExecuteUnityCommand(string extraArgs, string projectPath = ".")
         {
             unityProcess.WaitForExit();
             result = unityProcess.GetExitCode();
-            if (logProcess.WaitForExit(0) && (logProcess.GetExitCode() != 0))
+            if (logProcess.WaitForExit(1000) && (logProcess.GetExitCode() != 0))
```

Comment:
What is this change?

Reply:
It's picked from my branch. The problem was - the log process didn't contain the last error. It was killed to fast. In addition we delete log file, so the error message was completely missed even local (and for sure on CI)

Reply:
https://github.com/microsoft/appcenter-sdk-unity/commit/8d5f8ce5c455f8213017ca6b5d1e02cdf3c63c61

Reply:
@annakocheshkova please exclude unrelated change from this PR. Let's fix this separately